AnagramSolver.Tests/AnagramSolverTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using NUnit.Framework;
+
+namespace AnagramSolver.Tests
+{
+    [TestFixture]
+    class AnagramSolverTests
+    {
+        private BusinessLogic.Classes.AnagramSolver _anagramSolver;
+        public const string _filePath = "C:\\Users\\jonas.jaugelis\\source\\repos\\AnagramSolver\\zodynas.txt";
+
+        [SetUp] public void Setup()
+        {
+            _anagramSolver = new BusinessLogic.Classes.AnagramSolver();
+        }
+
+        [Test]
+        [TestCase("sula", "alus")]
+        [TestCase("alus", "sula")]
+        [TestCase("mirti", "rimti")]
+        [TestCase("rimti", "mirti")]
+        public void Should_ReturnOneAnagram_When_GivenOneWord(string command, string anagram)
+        {
+            var allAnagrams = _anagramSolver.GetAnagrams(command, 4, @_filePath);
+
+            Assert.That(allAnagrams[0].Word, Is.EqualTo(anagram));
+        }
+
+        [Test]
+        [TestCase("balos")]
+        public void Should_ReturnAllAnagrams_When_GivenOneWord(string command)
+        {
+            var allAnagrams = _anagramSolver.GetAnagrams(command, 4, @_filePath);
+
+            Assert.That(allAnagrams.Count, Is.GreaterThan(1));
```

Comment:
Discussed in meeting about different Assert's methods. They are indifferent, and next syntax is very welcoming. However it's good to know classic syntax as well. In this example `Assert.Greater(actualcount, 1);` would be considered classic syntax. In some cases this is easier to understand what is actual and what is expected value.